.bashrc

```diff
@@ -112,6 +112,35 @@ if ! shopt -oq posix; then
   if [ -f /usr/share/bash-completion/bash_completion ]; then
     . /usr/share/bash-completion/bash_completion
   elif [ -f /etc/bash_completion ]; then
-    . /etc/bash_completion  
+    . /etc/bash_completion
   fi
 fi
+
+rhino-up() {
+
+    local updateScript="${HOME}/rhinoupdate/script/update.sh"
+
+    mkdir -p ~/rhinoupdate/{kernel,script}
+
+    git clone https://github.com/rollingrhinoremix/rhino-update ~/rhinoupdate/script
+
+    # Set permissions to execute script.
+    [[ -x "${updateScript}" ]] || chmod +x "${updateScript}"
+
+    # Run script.
+    "${updateScript}"
+
+    rm -rf ~/rhinoupdate
+}
+
+rhino-in() {
+
+    mkdir -p ~/.rhino/{updates,config}
+
+    for rhinoFile in ~/.rhino/updates/configuration ~/.rhino/updates/config-v2; do
+
+        : >  "${rhinoFile}"
+    done
+
+    sudo apt update && sudo apt upgrade -y
+}
```

Comment:
Should not be in `.bashrc` but rather in `.bash_aliases` as to keep the two files separate when the time comes to update them. Is this possible?

Reply:
It is possible, however putting functions in an **aliases** file doesn't really make sense.

Reply:
The second option seems the most sensible. I wasn't asking to put it into the `.bash_aliases` file more than I was saying to do something similar to that.

Reply:
Ok then, I will create a .bash_functions file and overwrite the commit history.

I am currently out, I'll mention you when I finish.

Reply:
Thank you

Reply:
Done @Chadano!